groovy/postInit/components/Batteries.groovy

```diff
@@ -111,14 +102,6 @@ BR.recipeBuilder()
         .EUt(30)
         .buildAndRegister()
 
-BCR.recipeBuilder()
```

Comment:
no point in removing battery recipes here since this is not the battery overhaul PR. merging this prematurely would make some batteries unobtainable

Reply:
also you didnt remove the materials either.

Reply:
They are moved to Solvents.groovy, not removed!